instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/test/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/SpringWebfluxServerInstrumentationTest.java

```diff
@@ -39,7 +41,7 @@ protected void configure(HttpServerTestOptions options) {
 
     options.setExpectedHttpRoute(
         endpoint -> {
-          if (endpoint == ServerEndpoint.PATH_PARAM) {
+          if (endpoint == PATH_PARAM) {
```

Comment:
Same change everywhere else:
```suggestion
          if (PATH_PARAM.equals(endpoint)) {
```